services/server/server.go

```diff
@@ -432,19 +433,35 @@ func LoadPlugins(ctx context.Context, config *srvconfig.Config) ([]*plugin.Regis
 				snapshotters[name] = sn.(snapshots.Snapshotter)
 			}
 
-			shared := true
-			ic.Meta.Exports["policy"] = srvconfig.SharingPolicyShared
+			contentShared := true
+			ic.Meta.Exports["content_policy"] = srvconfig.SharingPolicyShared
```

Comment:
are the `Exports` anywhere, or just when running `ctr plugins ls -d`?
also, minor nit, should it be `contentPolicy`, to be more go-like?